src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpGoToDefinition.cs

```diff
@@ -69,26 +69,30 @@ public void GoToDefinitionOpensProvisionalTabIfDocumentNotAlreadyOpen()
     SomeClass sc;
 }");
             VisualStudio.Editor.PlaceCaret("SomeClass");
-            VisualStudio.Editor.GoToDefinition();
+            VisualStudio.Editor.GoToDefinition("FileDef.cs");
             VisualStudio.Editor.Verify.TextContains(@"class SomeClass$$", assertCaretPosition: true);
             Assert.True(VisualStudio.Shell.IsActiveTabProvisional());
         }
 
         [WpfFact, Trait(Traits.Feature, Traits.Features.GoToDefinition)]
```

Comment:
So I would have expected xUnit would see this method through the inherited class as well, and then in the inherited copy see this method with LSP trait, so it'd run this in the LSP case as well. If this works it works, but not entirely sure what might happen here.